models/location.go

```diff
@@ -0,0 +1,32 @@
+package models
+
+import (
+	"context"
+	"database/sql"
+	"log"
+)
+
+type Location struct {
+	Id      int    `json:"id"`
+	Name    string `json:"name"`
+	Address string `json:"address"`
+	URL     string `json:"url"`
```

Comment:
Since we're still finalizing the tables and have not deployed it yet I updated the name in the original create table statement. I called it `map_url` since it may be from other providers other than google.